script.js

```diff
@@ -1,21 +1,299 @@
+['ShiftLeft', 'KeyZ', 'KeyX', 'KeyC', 'KeyV', 'KeyB', 'KeyN', 'KeyM', 'Comma', 'Period', 'Slash', 'ArrowUp', 'ShiftRight'],
+['ControlLeft', 'MetaLeft', 'AltLeft', 'Space', 'AltRight', 'ArrowLeft', 'ArrowDown', 'ArrowRight', 'ControlRight']];
+
 
 let container = document.createElement('div');
 container.className = 'container';
 document.body.append(container);
-let textArea=document.createElement('textarea');
+let textArea = document.createElement('textarea');
 textArea.className = 'textarea-style';
-container.appendChild   (textArea);    
+container.appendChild(textArea);
+
+//create buttons
+function createkeyboard(inerText) {
+    for (let i = 0; i < inerText.length; i++) {
+        let line = document.createElement('divv');
+        line.classList.add("button-line-style");
+        container.appendChild(line);
+        for (let j = 0; j < inerText[i].length; j++) {
+            let btn = document.createElement('div');
+            btn.classList.add(style(keyCode[i][j]));
+            btn.id = keyCode[i][j];
+            btn.innerText = inerText[i][j];
+            line.appendChild(btn);
+        }
+
+    }
+}
+
+// change div inerText
+function refresh(str) {
+    if (str == "rus") {
+        container.querySelectorAll('div').forEach(function (e, i) {
+            e.innerText = Keyboard.inerText.rus.flat()[i];
+        });
+    }
+    if (str == "eng") {
+        container.querySelectorAll('div').forEach(function (e, i) {
+            e.innerText = Keyboard.inerText.eng.flat()[i];
+        });
+    }
+    if (str == "shift") {
+
+        container.querySelectorAll('div').forEach(function (e, i) {
+            if (e.innerText.length == '1') {
+                e.innerText = e.innerText.toUpperCase();
+            }
+
+            if (localStorage.getItem("lang") == "rus") {
+                if (Keyboard.spec_symbol.rus[e.id] != undefined) {
+                    e.innerText = Keyboard.spec_symbol.rus[e.id];
+                }
+            }
+            if (localStorage.getItem("lang") == "eng") {
+                if (Keyboard.spec_symbol.eng[e.id] != undefined) {
+                    e.innerText = Keyboard.spec_symbol.eng[e.id];
+                }
+            }
+
+
+        });
+    }
+    if (str == "shift-off") {
+        if (localStorage.getItem("lang") == "rus") { refresh('rus'); }
+        if (localStorage.getItem("lang") == "eng") { refresh('eng'); }
+
+    }
+
+}
+
+//button_style
+function style(keyCode) {
+    if (keyCode == "Backspace" || keyCode == "ShiftRight" || keyCode == "CapsLock") return ("button-style-v1");
+    else if (keyCode == "Tab" || keyCode == "Enter" || keyCode == "ShiftLeft") return ("button-style-v2");
+    else if (keyCode == "Space") return ("button-style-v3");
+    else if (keyCode == "ControlLeft" || keyCode == "MetaLeft" || keyCode == "AltLeft" || keyCode == "ControlRight" || keyCode == "AltRight" || keyCode == "Delete") return ("button-style-v4");
+    else return ("button-style");
+}
```

Comment:
it's work correct, but for styles better use css, also because you use object, you can save style inside it like a field